components/script/timers.rs

```diff
@@ -419,7 +419,7 @@ enum InternalTimerCallback {
 }
 
 impl JsTimers {
-    pub fn new() -> JsTimers {
+    pub fn default() -> JsTimers {
```

Comment:
This method should also be the implementation of the `Default` trait. Here's an example of this from the rust documentation: https://doc.rust-lang.org/std/default/trait.Default.html#how-can-i-implement-default